woocommerce/woocommerce-android
# Hide mark order complete button unpaid orders

Closes: #6894 


### Description


This PR tweaks the visibility of the `MarkOrderCompleteButton` so it's hidden for all unpaid orders and for orders that already have the COMPLETE status

### Testing instructions


1. Go to the Orders list
2. Open an order that has the COMPLETE status
3. Make sure the Mark Order Complete button is not showing
4. open other orders with different statuses and confirm the button visibility ( if the order is unpaid the button should be hidden is the order is paid - meaning it shows a pay date) the button should be displayed - unless the status is COMPLETE


### Images/gif


<img src="https://user-images.githubusercontent.com/30724184/178795541-c136ab77-4db4-40e7-9d8a-7647b7363157.gif" width="350"/> |
